resources/js/store/auth/login/index.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  namespaced: true,
+  state() {
+    return { errors: {} };
+  },
+  mutations: {
+    login(state, userData) {
```

Comment:
This is wrong. Mutations should only have logic to mutate the state data and nothing else. The axios calls should be written in actions and when the axios call succeeds, the action will commit the data using mutation. Make this change everywhere.

Reply:
ok